platform/linuxbsd/display_server_x11.cpp

```diff
@@ -742,6 +742,27 @@ Size2i DisplayServerX11::screen_get_size(int p_screen) const {
 	return screen_get_usable_rect(p_screen).size;
 }
 
+float DisplayServerX11::screen_get_refresh_rate(int p_screen) const {
+	_THREAD_SAFE_METHOD_
+
+	if (p_screen == SCREEN_OF_MAIN_WINDOW) {
+		p_screen = window_get_current_screen();
+	}
+
+	//invalid screen?
+	ERR_FAIL_INDEX_V(p_screen, get_screen_count(), 0);
+
+	float refresh_rate = 60.0; //Default to 60 hz if the engine was unable to get the refresh rate.
```

Comment:
Is there a place you can store the default value (maybe in `display_server.h`) and just reference it?

Reply:
I would do that, but it looks like most other places that have a default value do it hard coded. Plus, I don't see many benefits to adding it to display_server.h since it most likely won't be changed any time soon, and if it is, it would be a very quick change.